### ISSUE TITLE
medical director bug (???)

### ISSUE THREAD

**Issue by Retrino** [goonstation/goonstation#3385]
So, for context, I have no idea what triggered this. Never seen nor heard of it happening before

Round startup, spawn as MD on rev round in cog1. Except, don't spawn in the MD's office. Spawn somewhere completely remote, in a sliced up version of an MD's office, a random hole in the floor, with a medibot that tells me "Delicious." for some reason. I almost mistook it for donut 2 or clarion's office, but it had slight differences.

Leaving the office was endless shadow and space. Walls examined as nothing, only a few actually acted as walls to bounce off of to continue space drifting. There were a couple areas that were lit on the other side of the walls.

Basically, I went to hell and I don't know why. Reporting it because... yes

edit;;; emily said it's z 4. still a bug though? i guess

**Comment by Tarmunora** [comment]
fixed
